src/libsystemd/sd-netlink/netlink-message.c

```diff
@@ -686,7 +685,8 @@ static int netlink_message_read_internal(sd_netlink_message *m, unsigned short t
 
         rta = (struct rtattr*)((uint8_t *) m->hdr + attribute->offset);
 
-        *data = RTA_DATA(rta);
+        if (data)
+                *data = RTA_DATA(rta);
```

Comment:
maybe return the two params ret_data and ret_net_byteorder even

Reply:
Hm? Sorry, what do you mean?

Reply:
Ah, you mean that the variables should be renamed, don't you?

Reply:
yupp. 

meant to write "rename" when i wrote "return". sorry for the confusion